src/mgr/ActivePyModules.cc

```diff
@@ -876,7 +880,77 @@ PyObject* ActivePyModules::with_perf_counters(
   return f.get();
 }
 
-PyObject* ActivePyModules::get_counter_python(
+PyObject* ActivePyModules::with_perf_counters(
+    std::function<void(PerfCounterInstance& counter_instance, PerfCounterType& counter_type, PyFormatter& f)> fct,
+    const std::string &svc_name,
+    const std::string &svc_id,
+    const std::string_view &counter_name,
```

Comment:
string_view is always received (in the function signature) by value.
I.e. std::string x.